README.md

```diff
@@ -13,13 +13,20 @@
   * [Paciello Group](https://developer.paciellogroup.com/resources/)
   * [Ebay accessibility patterns](https://ebay.gitbooks.io/mindpatterns/content/)
   * [BBC accessibility guidelines](http://www.bbc.co.uk/guidelines/futuremedia/accessibility/)
+  * [Vox Media Accessibility Checklist](http://accessibility.voxmedia.com/)
 
-## Blogs, etc.
+## Other Resources (Blogs, Individuals, Videos, etc)
 
   * [24 Accessibility](https://www.24a11y.com/) - series of articles on digital accessibility
-  * [Leonie Watson](https://tink.uk/)
-  * [Scott O'Hara](https://www.scottohara.me/all-writings/)
+  * [Leonie Watson](https://tink.uk/) - Leading Accessibility engineer, on W3C Web Platform Working Group. [@LeonieWatson](https://twitter.com/LeonieWatson) 
+  * [Scott O'Hara](https://www.scottohara.me/all-writings/) UX Accessibility at Paciello Group. [@scottohara](https://twitter.com/scottohara)
```

Comment:
Also for consistency - add hyphen after the link, lowercase `Accessibility`, and remove period